config/initializers/slack.rb

```diff
@@ -3,5 +3,9 @@
 end
 
 Slack::Events.configure do |config|
-  config.signing_secret = ENV["SLACK_SIGNING_SECRET"]
+  config.signing_secret = if Rails.env.test?
+    "FICTIONAL63fZabecd7c3c89387b6a9X"
```

Comment:
**question/suggestion**: Do the tests require this specific value due to how things are signed? I mainly ask because this would introduce a magical value that doesn't mean anything specific to anyone coming to read the code. Alternatively, if it's strictly for tests, then having this in the test file(s)
```
allow(Slack::Events.config).to receive(:signing_secret) { SecureRandom.uuid }
```
would be a clearer picture and communicate that the value doesn't matter.

If this value does matter, a comment might be helpful.

Reply:
> Do the tests require this specific value due to how things are signed?

No, it just needs _some_ value. Tests need to use the value to create the HMAC signature when sending requests but it's not reliant on that specific value for anything.